crates/jstz_runtime/src/ext/mod.rs

```diff
@@ -1,6 +1,9 @@
 pub(crate) mod jstz_console;
 pub(crate) mod jstz_fetch;
 pub mod jstz_kv;
+
+pub mod jstz_test;
+
```

Comment:
```suggestion
pub mod jstz_test;
```
Can we remove the empty lines?